.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
         -   id: end-of-file-fixer
         -   id: trailing-whitespace
     -   repo: https://github.com/psf/black
-        rev: 22.1.0
+        rev: 22.3.0
```

Comment:
Just handled this in #1052. Mind reverting the pre-commit changes here?

Reply:
Done

Reply:
Thanks :+1: